ci-operator/step-registry/image-based-install-operator/gather/image-based-install-operator-gather-commands.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ image based install operator gather command ************"
+
+source "${SHARED_DIR}/packet-conf.sh"
+
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << "EOF"
+
+# prepending each printed line with a timestamp
+exec > >(awk '{ print strftime("[%Y-%m-%d %H:%M:%S]"), $0 }') 2>&1
+
+set -xeo pipefail
+
+cd /root/dev-scripts
+source common.sh
+source utils.sh
+source network.sh
+
+mkdir /root/ibio-gather
+oc get baremetalhost ostest-extraworker-0 -n openshift-machine-api -o yaml > /root/ibio-gather/baremetalhost.yaml
```

Comment:
Will all the BMHs exist in this namespace? Maybe just get all of them (`--all-namespaces`)?

Reply:
They should all exist in openshift-machine-api, including the BMHs for the hub cluster's nodes. If we use --all-namespaces then we can't specify this specific BMH that we care about, and then the file will include 6 extra BMHs that I think will make the baremetalhost.yaml file less useful.

Reply:
If we're definitely creating the BMH there then that's fine with me. I only asked because I think we switch the namespace of the BMH for the assisted ZTP tests.

Reply:
The yaml that we use to create the bmh and it's secret are created fot us by dev-scripts, and we don't touch the namespace. Also I think by default the bmh operator does not watch all namespaces, just openshift-machine-api.